src/styles/header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  max-width: 1200px;
+  display: flex;
+  font-family: Roboto, system-ui;
```

Comment:
It is recommended to define the font-family in a more global scope, such as the root or body element. This way, you don't have to repeat it in multiple places and it's easier to change if needed.